src/pnet/key-generator.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const crypto = require('crypto')
+const randomBytes = require('iso-random-stream/src/random')
```

Comment:
Why not just use `libp2p-crypto` random bytes? It uses the same thing under the hood and avoids us needing to worry about another dependency since we have to use `libp2p-crypto` anyway.